context/deploy.js

```diff
@@ -0,0 +1,21 @@
+//Generated  file 
+'use strict';
+module.exports = {
+    payloadJsonSchema: {
```

Comment:
Yeah for sure. If we know what headers we want then we can add them into the context